src/gdal/ogr_file_format_p.h

```diff
@@ -272,6 +297,50 @@ MapCoord OgrFileImport::toMapCoord(double x, double y) const
 	return (this->*to_map_coord)(x, y);
 }
 
+/**
+ * An Exporter to geospatial vector data supported by OGR.
+ *
+ * OGR needs to know the filename. The filename is derived from
+ * a QFile passed as stream to the constructor.
+ */
+class OgrFileExport : public Exporter
+{
+	Q_DECLARE_TR_FUNCTIONS(OpenOrienteering::OgrFileExport)
+
+public:
+	OgrFileExport(QIODevice* stream, Map *map, MapView *view);
+	~OgrFileExport() override {}
+
+	void doExport() override;
+
+	bool noIoStream() { return true; }
```

Comment:
Just for completeness, it needs to be marked `override` here. And the implementation should go to the cpp file. (It is a virtual function, and not on a critical path.)

However I don't think it will remain like this. The name has no analogy in Mapper, and we already have flags for describing capabilities in `FileFormat`.